tests/test_inverses.py

```diff
@@ -136,6 +130,27 @@ def circuit():
 
         assert np.allclose(circuit(), expected_output, atol=tol, rtol=0)
 
+    @pytest.mark.parametrize("name,par,expected_output", [
+        ("Rot", [math.pi/2, 0, 0], 1),
+        ("Rot", [0, math.pi/2, 0], 0),
+        ("Rot", [0, 0, math.pi/2], 1),
+        ("Rot", [math.pi/2, -math.pi/4, -math.pi/4], 1/math.sqrt(2)),
+        ("Rot", [-math.pi/4, math.pi/2, math.pi/4], 0),
+        ("Rot", [-math.pi/4, math.pi/4, math.pi/2], 1/math.sqrt(2)),
+    ])
+    def test_unsupported_gate_inverses(self, name, par, expected_output):
```

Comment:
Reorganized this into a separate test case, as ``assert dev.supports_operation("Rot")`` evaluates to False